dask/array/overlap.py

```diff
@@ -637,6 +637,15 @@ def map_overlap(
         trim = get(sig.index("trim"), args, trim)
         func, args = args[0], [func]
 
+    if boundary is None:
+        # Default boundary value is set in the function named "coerce_boundary"
+        warnings.warn(
+            "Default 'boundary' argument value will change from 'reflect' "
+            "to 'none' in future versions. "
```

Comment:
Should we specify a version here?

Reply:
I think it can be relatively soon. Maybe February?

Reply:
I'd like to specify a version. 

February seems too early though, lots of people will be on holiday in December and January, so it's quite likely people won't see this at all before it changes. (Although I'm sure plenty of scientists have code they run infrequently, like once every six months or so on a new dataset, so maybe it's impractical to wait a huge amount of time.)

Mind you, the only visible differences should be at the very outer edges of a Dask array, so the impact of the change is pretty small. Maybe it's ok to change it soon and have people's first introduction be a pleasant surprise things are running faster.

Reply:
We can always change the version if we don't feel ready. But I tend to agree that I don't think this will negatively impact many people. From the responses on this, the `FutureWarning` won't really hit them either since they set this kwarg. So I'm fine with whatever you choose.

Reply:
Ok, I'll say March then, `2021.03.0`